test/fixtures/shared/variables.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "parent_id" {
+  description = "The parent of this AccessPolicy in the Cloud Resource Hierarchy. As of now, only organization are accepted as parent."
+}
+
+variable "policy_name" {
+  description = "The policy's name."
+}
+
+variable "protected_project_ids" {
+  description = "Project id and number of the project within the regular service perimeter"
+  type        = "map"
+
+  default {
```

Comment:
We shouldn't include defaults if they are not valid values. The description of a map should include any expected keys and values.

Reply:
Fixed in 92af66c0d2149763da51858bc859b4e7cdb902b8